src_base/xevd_ipred.h

```diff
@@ -60,5 +60,5 @@ void ipred_vert(pel *src_le, pel *src_up, pel * src_ri, u16 avail_lr, pel *dst,
 void ipred_plane(pel *src_le, pel *src_up, pel *src_ri, u16 avail_lr, pel *dst, int w, int h, int bit_depth);
 void ipred_bi(pel *src_le, pel *src_up, pel *src_ri, u16 avail_lr, pel *dst, int w, int h, int bit_depth);
 void ipred_ang(pel *src_le, pel *src_up, pel *src_ri, u16 avail_lr, pel *dst, int w, int h, int ipm, int bit_depth);
-int  intra_mode_list[IPD_CNT];
+//int  intra_mode_list[IPD_CNT];
```

Comment:
could you remove "intra_mode_list" line?

Reply:
I remove "intra_mode_list".